pkg/machines/components/infoRecord.jsx

```diff
@@ -21,20 +21,37 @@ import React from "react";
 import PropTypes from "prop-types";
 import { OverlayTrigger, Tooltip } from "patternfly-react";
 
+import cockpit from "cockpit";
+
+const _ = cockpit.gettext;
+
 const InfoRecord = ({ id, descr, value, descrClass, valueClass, tooltip }) => {
-    return (<tr>
-        <td className={descrClass || 'top'}>
-            <label className='control-label'>
-                {descr}
-            </label>
-        </td>
-        <td id={id} className={valueClass}>
-            {value}
-        </td>
-        {tooltip && (<td><OverlayTrigger overlay={ <Tooltip id="tip-inforec">{tooltip}</Tooltip> } placement="top">
-            <span className="fa fa-lg fa-info-circle" />
-        </OverlayTrigger></td>)}
-    </tr>);
+    let labelClass = cockpit.format(_("control-label $0"), descrClass || 'top');
+    let infoContent;
+
+    if (tooltip) {
+        infoContent = (
+            <div id={id} className={valueClass} role="group">
+                {value}
+                {tooltip && (<OverlayTrigger overlay={ <Tooltip id="tip-inforec">{tooltip}</Tooltip> } placement="top">
+                    <span className="fa fa-lg fa-info-circle" />
+                </OverlayTrigger>)}
+            </div>
+        );
+    } else {
+        infoContent = (
+            <div id={id} className={valueClass} role="group">
```

Comment:
For consistency, I also made InfoRecords without tooltips act the same as those with tooltips. So it's just a group of 1. It's probably not needed and overkill, but hey, better than acting different and also having a wrapping div, right?